apps/logo-quiz/src/app/views/LogoVerify/LogoVerify.tsx

```diff
@@ -27,8 +30,11 @@ interface LogoVerifyProps extends RouteComponentProps<MatchParams> {
   removeLetterFromGuess: typeof removeLetterFromGuess;
   fetchLogo: typeof fetchLogo;
   flushLogo: typeof flushLogo;
-  guess: GuessedLetter[];
+  validateLogo: typeof validateLogo;
```

Comment:
what is this typeof doing here?

Reply:
We need a way to tell the component "the function 'validateLogo' is valid, you can use it". We do it by creating this interface `LogoVerifyProps`. As `validateLogo` is a function itself, we cannot use it as a type like `validateLogo: validateLogo`. Therefore, we need to tell the component "you can use `validateLogo` which is `typeof validateLogo`".

I know it's super confusing like that. Read this for full explanation.